### ISSUE TITLE
Lane/Tool offset w/ Nearest waypoint

### ISSUE THREAD

**Issue by DynamicXYZ** [Courseplay/courseplay#1831]
I'm trying to make 2 combines run a field course and when approaching the field the harvesters veer off course from the offset I gave them and then they start going back on course. This happens every single time and it's preventing me from harvesting the field since it leaves behind large section of unharvested field.

This happens when I start the helper at "nearest waypoint" at the beginning of the field course lanes.

[Picture](https://drive.google.com/drive/folders/0ByuaHC6Vp2QbS0ROMS1nTVNhWHc?usp=sharing)

**Comment by Raints** [comment]
So you start course from right spot and "Start course at first waypoint"? What I could try first is to lower the field speed to 10 km/h. Then it will drive to start a bit slower. Just try, not sure it will help you.

**Comment by DynamicXYZ** [comment]
Using nearest waypoint doesn't work well when I start the helper. I have to use first waypoint and its annoying because sometimes I have to quit the game when my field is half way finished and when I come back on it has to start all over again...

**Comment by pops64** [comment]
Try Starting further back or more aligned with where you think CP wants to start the harvester. Or starting where it works for first waypoint

**Comment by DynamicXYZ** [comment]
That still doesn't solve the issue that I am having with starting at "nearest waypoint".

**Comment by nyfarmboy07** [comment]
Typically when I run two machines I create a course with machine #1 with the course width being the total width of both machines.  Then, I offset the tool of machine 1 to the right and machine 2 to the left.  Once machine 1 is running on course, I start machine 2 by using the copy course function.  This makes it so both machines are running on the same exact course.  

Always start field work or seeding at the first waypoint.  If you stop in the middle of the field, then switching to nearest waypoint should allow the machine to continue back on course.  I show the course points when doing this because often, the nearest point may not be the one directly in front of the machine, but to the side of a parallel track.  This happens a lot when using machines under 4m in width.

**Comment by DynamicXYZ** [comment]
Yeah I changed it to first waypoint when I started the field but let's say you stop the worker at one end of the field lane and start and re-position them for the next lane using nearest waypoint. Half the time it'll go on without an issue. Some maps have really big fields and I just can't simply afford to just start at the first waypoint again. The only thing that works is the start the course at the end of the previous lane before it has to turn around.
